website/notifications/emails.py

```diff
@@ -124,6 +126,11 @@ def check_node(node, event):
     return node_subscriptions
 
 
+def check_target_user(user, event):
```

Comment:
This is a misnomer: "check_" implies that the function will return a boolean. How about `filter_by_user_and_event`?